{"DBCallId":"1b893163-407f-40a4-abc1-20feec55cc2b","TimeStamp":"2024-12-16T14:45:16.4837151-03:00","Query":" SELECT DATA_MOV_ABERTO , DATA_MOV_ABERTO FROM SEGUROS.SISTEMAS WHERE IDE_SISTEMA = ' ' WITH UR","sqlCode":999,"rowsCount":0,"Time":"1 ms","Error":">>> Erro: ExecuteReader requires an open connection. The connection's current state is Closed."}
{"DBCallId":"bb5f55b3-db8f-4c36-a467-e9780a2b2794","TimeStamp":"2024-12-16T14:45:19.7290214-03:00","Query":" SELECT VALUE(MAX(NSAS_SIVPF) ,0) FROM SEGUROS.ARQUIVOS_SIVPF WHERE SIGLA_ARQUIVO = ' ' AND SISTEMA_ORIGEM = '0000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"39656c7b-6662-4d24-b9eb-3292f81fd1d5","TimeStamp":"2024-12-16T14:45:21.0487685-03:00","Query":" SELECT VALUE(MAX(NSAS_SIVPF) ,0) FROM SEGUROS.ARQUIVOS_SIVPF WHERE SIGLA_ARQUIVO = ' ' AND SISTEMA_ORIGEM = '0000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"1 ms","Error":""}
{"DBCallId":"14694c5b-49e2-45eb-bdb6-f6f9deee95ca","TimeStamp":"2024-12-16T14:45:42.0675859-03:00","Query":"SELECT A.NUM_TERMO , A.COD_SUBGRUPO , A.DATA_ADESAO , A.COD_AGENCIA_OP , A.NUM_MATRICULA_VEN , A.CODPDTVEN , A.PCCOMVEN , A.PCADIANTVEN , A.COD_AGENCIA_VEN , A.OPERACAO_CONTA_VEN , A.NUM_CONTA_VEN , A.DIG_CONTA_VEN , A.NUM_MATRICULA_GER , A.CODPDTGER , A.PCCOMGER , A.COD_AGENCIA_GER , A.OPERACAO_CONTA_GER , A.NUM_CONTA_GER , A.DIG_CONTA_GER , A.NUM_MATRICULA_SUR , A.CODPDTSUR , A.PCCOMSUR , A.NUM_MATRICULA_GCO , A.CODPDTGCO , A.PCCOMGCO , A.MODALIDADE_CAPITAL , A.TIPO_PLANO , A.IND_PLANO_ASSOCIAD , A.COD_PLANO_VGAPC , A.COD_PLANO_APC , A.VAL_CONTRATADO , A.VAL_COMISSAO_ADIAN , A.QUANT_VIDAS , A.TIPO_COBERTURA , A.PERI_PAGAMENTO , A.TIPO_CORRECAO , A.PERIODO_CORRECAO , A.COD_MOEDA_IMP , A.COD_MOEDA_PRM , A.COD_CLIENTE , A.OCORR_ENDERECO , A.COD_CORRETOR , A.PCCOMCOR , A.COD_ADMINISTRADOR , A.PCCOMADM , A.COD_USUARIO , A.DATA_INCLUSAO , A.SITUACAO , A.NUM_PROPOSTA , A.NUM_RCAP , A.DATA_RCAP , A.VAL_RCAP , A.NUM_APOLICE , B.NUM_PROPOSTA_SIVPF , B.DTH_DIA_DEBITO , B.COD_AGENCIA_DEB , B.OPERACAO_CONTA_DEB , B.NUM_CONTA_DEBITO , B.DIG_CONTA_DEBITO , B.DTH_LIBERACAO FROM SEGUROS.TERMO_ADESAO A , SEGUROS.VG_COMPL_TERMO B WHERE A.SITUACAO = '0' AND B.NUM_TERMO = A.NUM_TERMO AND B.DTH_LIBERACAO = '2020-01-01'","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"2c5d55fb-d65d-40ca-bd02-7d0b85f478b4","TimeStamp":"2024-12-16T14:45:43.4876301-03:00","Query":"SELECT BCO_AVISO , AGE_AVISO , NUM_AVISO_CREDITO , DATA_MOVIMENTO , DATA_RCAP , COD_OPERACAO FROM SEGUROS.RCAP_COMPLEMENTAR WHERE COD_FONTE = '0000' AND NUM_RCAP = '000000000' AND NUM_RCAP_COMPLEMEN = 0 ORDER BY COD_OPERACAO DESC","sqlCode":0,"rowsCount":3,"Time":"0 ms","Error":""}
{"DBCallId":"dc5cd910-786e-4b0a-8776-07ff38c5b446","TimeStamp":"2024-12-16T14:46:04.9324099-03:00","Query":" SELECT DISTINCT COD_EMPRESA_SIVPF , COD_RELAC FROM SEGUROS.PRODUTOS_SIVPF WHERE COD_EMPRESA_SIVPF = 1 AND COD_PRODUTO_SIVPF = '0000' ORDER BY COD_EMPRESA_SIVPF , COD_RELAC WITH UR","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"751aa64c-13f8-40d0-9f73-d0c9064b389c","TimeStamp":"2024-12-16T14:46:11.3021111-03:00","Query":" SELECT COD_FONTE , NUM_PROPOSTA , VAL_RCAP , VAL_RCAP_PRINCIPAL , DATA_CADASTRAMENTO , DATA_MOVIMENTO , SIT_REGISTRO , COD_OPERACAO , NUM_TITULO , AGE_COBRANCA , NUM_RCAP FROM SEGUROS.RCAPS WHERE NUM_CERTIFICADO = '000000000000000' WITH UR","sqlCode":999,"rowsCount":0,"Time":"1 ms","Error":">>> Erro: ERROR [58005] [IBM][DB2.NET] SQL0902 An unexpected exception has occurred in Process: 13156 Thread 28 AppDomain: Name:testhost There are no context policies. Function: DB2DataReader.Read() CallStack: at System.Environment.get_StackTrace() at IBM.Data.Db2.DB2ConnPool.HandleUnknownErrors(String strFncMsg, Exception exception, Boolean bThrow) at IBM.Data.Db2.DB2DataReader.Read() at Dapper.SqlMapper.QueryImpl[T](IDbConnection cnn, CommandDefinition command, Type effectiveType)+MoveNext() in /_/Dapper/SqlMapper.cs:line 1201 at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection) at Dapper.SqlMapper.Query(IDbConnection cnn, String sql, Object param, IDbTransaction transaction, Boolean buffered, Nullable`1 commandTimeout, Nullable`1 commandType) in /_/Dapper/SqlMapper.cs:line 737 at IA_ConverterCommons.QueryBasis`1.Open(Object procParams) in E:\\PROJETOS\\FOURSYS\\SIAS\\SIAS-Batch\\IA_ConverterCommons\\Basis\\QueryBasis.cs:line 234 at Sias.PessoaFisica.DB2.PF0642B.R0200_00_LER_RCAPS_DB_SELECT_1_Query1.Execute(R0200_00_LER_RCAPS_DB_SELECT_1_Query1 r0200_00_LER_RCAPS_DB_SELECT_1_Query1) in E:\\PROJETOS\\FOURSYS\\SIAS\\SIAS-Batch\\Sias.PessoaFisica\\DB2\\PF0642B\\R0200_00_LER_RCAPS_DB_SELECT_1_Query1.cs:line 95 at Code.PF0642B.R0200_00_LER_RCAPS_DB_SELECT_1() in E:\\PROJETOS\\FOURSYS\\SIAS\\SIAS-Batch\\Sias.PessoaFisica\\Code\\PF0642B.cs:line 2517 at FileTests.Test_DB.PF0642B_Tests_DB.PF0642B_Database() in E:\\PROJETOS\\FOURSYS\\SIAS\\SIAS-Batch\\FileTests.Test\\FileTests.Test\\PF0642B_Tests_DB.cs:line 37 at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor) at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr) at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters) at Xunit.Sdk.TestInvoker`1.CallTestMethod(Object testClassInstance) in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestInvoker.cs:line 151 at Xunit.Sdk.TestInvoker`1.<>c__DisplayClass48_0.<<InvokeTestMethodAsync>b__1>d.MoveNext() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestInvoker.cs:line 278 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestInvoker`1.<>c__DisplayClass48_0.<InvokeTestMethodAsync>b__1() at Xunit.Sdk.ExecutionTimer.AggregateAsync(Func`1 asyncAction) in /_/src/xunit.execution/Sdk/Frameworks/ExecutionTimer.cs:line 48 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.ExecutionTimer.AggregateAsync(Func`1 asyncAction) at Xunit.Sdk.TestInvoker`1.<>c__DisplayClass48_0.<InvokeTestMethodAsync>b__0() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestInvoker.cs:line 243 at Xunit.Sdk.ExceptionAggregator.RunAsync(Func`1 code) in /_/src/xunit.core/Sdk/ExceptionAggregator.cs:line 90 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.ExceptionAggregator.RunAsync(Func`1 code) at Xunit.Sdk.TestInvoker`1.InvokeTestMethodAsync(Object testClassInstance) in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestInvoker.cs:line 242 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestInvoker`1.InvokeTestMethodAsync(Object testClassInstance) at Xunit.Sdk.XunitTestInvoker.InvokeTestMethodAsync(Object testClassInstance) in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestInvoker.cs:line 112 at Xunit.Sdk.TestInvoker`1.<RunAsync>b__47_0() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestInvoker.cs:line 207 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestInvoker`1.<RunAsync>b__47_0() at Xunit.Sdk.ExceptionAggregator.RunAsync[T](Func`1 code) in /_/src/xunit.core/Sdk/ExceptionAggregator.cs:line 107 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.ExceptionAggregator.RunAsync[T](Func`1 code) at Xunit.Sdk.TestInvoker`1.RunAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestInvoker.cs:line 190 at Xunit.Sdk.XunitTestRunner.InvokeTestMethodAsync(ExceptionAggregator aggregator) in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestRunner.cs:line 84 at Xunit.Sdk.XunitTestRunner.InvokeTestAsync(ExceptionAggregator aggregator) in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestRunner.cs:line 67 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.XunitTestRunner.InvokeTestAsync(ExceptionAggregator aggregator) at Xunit.Sdk.TestRunner`1.<>c__DisplayClass43_0.<RunAsync>b__0() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestRunner.cs:line 149 at Xunit.Sdk.ExceptionAggregator.RunAsync[T](Func`1 code) in /_/src/xunit.core/Sdk/ExceptionAggregator.cs:line 107 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.ExceptionAggregator.RunAsync[T](Func`1 code) at Xunit.Sdk.TestRunner`1.RunAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestRunner.cs:line 149 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestRunner`1.RunAsync() at Xunit.Sdk.XunitTestCaseRunner.RunTestAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestCaseRunner.cs:line 139 at Xunit.Sdk.TestCaseRunner`1.RunAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestCaseRunner.cs:line 82 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestCaseRunner`1.RunAsync() at Xunit.Sdk.XunitTestCase.RunAsync(IMessageSink diagnosticMessageSink, IMessageBus messageBus, Object[] constructorArguments, ExceptionAggregator aggregator, CancellationTokenSource cancellationTokenSource) in /_/src/xunit.execution/Sdk/Frameworks/XunitTestCase.cs:line 162 at Xunit.Sdk.XunitTestMethodRunner.RunTestCaseAsync(IXunitTestCase testCase) in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestMethodRunner.cs:line 45 at Xunit.Sdk.TestMethodRunner`1.RunTestCasesAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestMethodRunner.cs:line 136 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestMethodRunner`1.RunTestCasesAsync() at Xunit.Sdk.TestMethodRunner`1.RunAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestMethodRunner.cs:line 106 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestMethodRunner`1.RunAsync() at Xunit.Sdk.XunitTestClassRunner.RunTestMethodAsync(ITestMethod testMethod, IReflectionMethodInfo method, IEnumerable`1 testCases, Object[] constructorArguments) in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestClassRunner.cs:line 199 at Xunit.Sdk.TestClassRunner`1.RunTestMethodsAsync() at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestClassRunner`1.RunTestMethodsAsync() at Xunit.Sdk.TestClassRunner`1.RunAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestClassRunner.cs:line 175 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestClassRunner`1.RunAsync() at Xunit.Sdk.XunitTestCollectionRunner.RunTestClassAsync(ITestClass testClass, IReflectionTypeInfo class, IEnumerable`1 testCases) in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestCollectionRunner.cs:line 185 at Xunit.Sdk.TestCollectionRunner`1.RunTestClassesAsync() at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestCollectionRunner`1.RunTestClassesAsync() at Xunit.Sdk.TestCollectionRunner`1.RunAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestCollectionRunner.cs:line 101 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestCollectionRunner`1.RunAsync() at Xunit.Sdk.XunitTestAssemblyRunner.RunTestCollectionAsync(IMessageBus messageBus, ITestCollection testCollection, IEnumerable`1 testCases, CancellationTokenSource cancellationTokenSource) in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestAssemblyRunner.cs:line 262 at Xunit.Sdk.XunitTestAssemblyRunner.<>c__DisplayClass14_2.<RunTestCollectionsAsync>b__2() in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestAssemblyRunner.cs:line 211 at System.Threading.Tasks.Task`1.InnerInvoke() at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread) at System.Threading.Tasks.Task.ExecuteEntry() at System.Threading.Tasks.SynchronizationContextTaskScheduler.<>c.<.cctor>b__8_0(Object s) at Xunit.Sdk.MaxConcurrencySyncContext.RunOnSyncContext(SendOrPostCallback callback, Object state) in /_/src/xunit.execution/Sdk/MaxConcurrencySyncContext.cs:line 106 at Xunit.Sdk.MaxConcurrencySyncContext.<>c__DisplayClass11_0.<WorkerThreadProc>b__0(Object _) in /_/src/xunit.execution/Sdk/MaxConcurrencySyncContext.cs:line 96 at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) at Xunit.Sdk.ExecutionContextHelper.Run(Object context, Action`1 action) in /_/src/xunit.execution/Sdk/Utility/ExecutionContextHelper.cs:line 110 at Xunit.Sdk.MaxConcurrencySyncContext.WorkerThreadProc() in /_/src/xunit.execution/Sdk/MaxConcurrencySyncContext.cs:line 96 at Xunit.Sdk.XunitWorkerThread.<>c.<QueueUserWorkItem>b__5_0(Object _) in /_/src/common/XunitWorkerThread.cs:line 37 at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread) InnerException Message: Year, Month, and Day parameters describe an un-representable DateTime. Check InnerException property for more detail. SQLSTATE=58005"}
{"DBCallId":"bf5bd81d-8cde-4b1f-84c0-b07121bae95e","TimeStamp":"2024-12-16T14:46:12.20999-03:00","Query":" SELECT BCO_AVISO , AGE_AVISO , NUM_AVISO_CREDITO , DATA_MOVIMENTO , DATA_RCAP FROM SEGUROS.RCAP_COMPLEMENTAR WHERE COD_FONTE = '0000' AND NUM_RCAP = '000000000' AND NUM_RCAP_COMPLEMEN = 0 AND COD_OPERACAO BETWEEN 100 AND 199 WITH UR","sqlCode":0,"rowsCount":1,"Time":"2 ms","Error":""}
{"DBCallId":"403d06d8-6a3d-4e56-a0d8-ed0a7158bfba","TimeStamp":"2024-12-16T14:46:43.3484371-03:00","Query":"SELECT COD_CLIENTE , COD_ENDERECO , OCORR_ENDERECO , ENDERECO , BAIRRO , CIDADE , SIGLA_UF , CEP , DDD , TELEFONE , FAX , TELEX , SIT_REGISTRO FROM SEGUROS.ENDERECOS WHERE COD_CLIENTE = '000000000' AND SIT_REGISTRO = '0' ORDER BY COD_ENDERECO","sqlCode":999,"rowsCount":0,"Time":"19 ms","Error":">>> Erro: ExecuteReader requires an open connection. The connection's current state is Closed."}
{"DBCallId":"38513314-ed93-45e3-bfa7-5e4623f0ee81","TimeStamp":"2024-12-16T14:46:44.2789584-03:00","Query":" SELECT SIT_PROPOSTA , NUM_PROPOSTA_SIVPF FROM SEGUROS.PROPOSTA_FIDELIZ WHERE NUM_PROPOSTA_SIVPF = '000000000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"e7f70f86-bad5-46b4-88ec-81664f455893","TimeStamp":"2024-12-16T14:46:45.0663091-03:00","Query":" SELECT COD_CLIENTE , NOME_RAZAO , TIPO_PESSOA , CGCCPF , SIT_REGISTRO , DATA_NASCIMENTO , COD_PORTE_EMP , COD_NATUREZA_ATIV , COD_RAMO_ATIVIDADE , COD_ATIVIDADE FROM SEGUROS.CLIENTES WHERE COD_CLIENTE = '000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"45784b07-08ef-47d9-882d-51349c409ca2","TimeStamp":"2024-12-16T14:46:45.8318495-03:00","Query":"SELECT NUM_CERTIFICADO , OCORR_HISTORICO , DATA_INIVIGENCIA , DATA_TERVIGENCIA , IMP_MORNATU , IMPMORACID , IMPINVPERM FROM SEGUROS.HIS_COBER_PROPOST WHERE (NUM_CERTIFICADO = '000000000000000' OR NUM_CERTIFICADO = '000000000000000') AND DATA_TERVIGENCIA = '2020-01-01'","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"57a77e91-d6e4-428c-9f57-879d3233095e","TimeStamp":"2024-12-16T14:46:52.5704536-03:00","Query":" SELECT COD_SUBGRUPO , COD_CLIENTE , CLASSE_APOLICE , COD_FONTE , TIPO_FATURAMENTO , FORMA_FATURAMENTO , FORMA_AVERBACAO , TIPO_PLANO , PERI_FATURAMENTO , PERI_RENOVACAO , PERI_RETROATI_INC , PERI_RETROATI_CAN , OCORR_ENDERECO , OCORR_END_COBRAN , BCO_COBRANCA , AGE_COBRANCA , DAC_COBRANCA , TIPO_COBRANCA , COD_PAG_ANGARIACAO , PCT_CONJUGE_VG , PCT_CONJUGE_AP , OPCAO_COBERTURA , OPCAO_CORRETAGEM , IND_CONSISTE_MATRI , IND_PLANO_ASSOCIA , SIT_REGISTRO , OPCAO_CONJUGE FROM SEGUROS.SUBGRUPOS_VGAP WHERE NUM_APOLICE = '0000000000000' AND COD_SUBGRUPO = '0000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"36 ms","Error":""}
{"DBCallId":"701642c5-50f6-464e-99b5-2d4cfba2c98c","TimeStamp":"2024-12-16T14:47:07.8177501-03:00","Query":" SELECT OPCAO_PAGAMENTO , PERI_PAGAMENTO , DIA_DEBITO , COD_AGENCIA_DEBITO , OPE_CONTA_DEBITO , NUM_CONTA_DEBITO , DIG_CONTA_DEBITO FROM SEGUROS.OPCAO_PAG_VIDAZUL WHERE (NUM_CERTIFICADO = '000000000000000' OR NUM_CERTIFICADO = '000000000000000') AND DATA_TERVIGENCIA = '2020-01-01' WITH UR","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"d8e0e00e-3c54-4a33-88d4-054ea8a344b0","TimeStamp":"2024-12-16T14:47:08.7203165-03:00","Query":"SELECT NUM_CERTIFICADO , VAL_COMISSAO_VG , VAL_COMISSAO_AP FROM SEGUROS.FUNDO_COMISSAO_VA WHERE (NUM_CERTIFICADO = '084654460000028' OR NUM_CERTIFICADO = '084654460000028') AND COD_OPERACAO = '1101'","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"a7797669-67db-4309-b875-8cb522d0266b","TimeStamp":"2024-12-16T14:47:09.5759866-03:00","Query":" SELECT DATA_QUITACAO , FAIXA_RENDA_IND , FAIXA_RENDA_FAM FROM SEGUROS.PROPOSTAS_VA WHERE NUM_CERTIFICADO = '000000000000000' OR NUM_CERTIFICADO = '084654460000028' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"f4807d7d-cd4c-4556-b73e-65aa5727e03c","TimeStamp":"2024-12-16T14:47:10.372917-03:00","Query":"SELECT NUM_PROPOSTA_SIVPF , DTH_INI_VIGENCIA , NUM_NIVEL_CARGO , DTH_FIM_VIGENCIA , IMP_SEGURADA , VLR_PRM_INDIVIDUAL , QTD_VIDAS FROM SEGUROS.VG_TERMO_NIVEL WHERE NUM_PROPOSTA_SIVPF = '000000000000000' AND DTH_INI_VIGENCIA <= '2020-01-01' AND DTH_FIM_VIGENCIA >= '2020-01-01' ORDER BY NUM_NIVEL_CARGO","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"316ab46a-23ab-4330-8383-b434fac7be5f","TimeStamp":"2024-12-16T14:47:11.123496-03:00","Query":" SELECT VAL_TARIFA FROM SEGUROS.TARIFA_BALCAO_CEF WHERE COD_EMPRESA = 0 AND NUM_MATRICULA = 9999999 AND TIPO_FUNCIONARIO = '5' AND (NUM_CERTIFICADO = '000000000000000' OR NUM_CERTIFICADO = '084654460000028') WITH UR","sqlCode":0,"rowsCount":2,"Time":"0 ms","Error":""}
{"DBCallId":"e94a02b5-ced2-4e74-8b66-bce3696f5582","TimeStamp":"2024-12-16T14:47:44.4330036-03:00","Query":" SELECT VALUE (SUM(VLR_PREMIO) ,0) FROM SEGUROS.VG_FUNC_COBERTURA WHERE NUM_PROPOSTA_SIVPF = '000000000000000' AND DTH_INI_VIGENCIA = '2020-01-01' AND COD_GARANTIA = '0000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"2bbc6844-69ad-44e0-a1fb-f57e726bfabf","TimeStamp":"2024-12-16T14:47:47.1917974-03:00","Query":"SELECT NUM_PROPOSTA_SIVPF , DTH_INI_VIGENCIA , NUM_CPF , DTH_FIM_VIGENCIA , NUM_NIVEL_CARGO , NUM_MATRICULA , NOM_FUNCIONARIO , DTH_NASCIMENTO , NUM_IDADE , STA_SEXO , STA_ESTADO_CIVIL , COD_PROFISSAO , IND_REPR_EMPRE , IND_IMP_DPS , DES_MOTIVO , NUM_DDD , NUM_TELEFONE , NUM_RG , COD_ORGAO_RG , COD_UF_ORGAO_RG , DTH_EMISSAO_RG , STA_FUNCIONARIO , COD_USUARIO FROM SEGUROS.VG_MOV_FUNCIONARIO WHERE NUM_PROPOSTA_SIVPF = '000000000000000' AND DTH_INI_VIGENCIA <= '2020-01-01' AND DTH_FIM_VIGENCIA >= '2020-01-01' AND STA_FUNCIONARIO = ' ' ORDER BY NUM_MATRICULA , DTH_INI_VIGENCIA","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"ea3bcb6c-ecf4-43cb-978c-705ee08a520e","TimeStamp":"2024-12-16T14:47:47.9576185-03:00","Query":"SELECT A.NUM_PROPOSTA_SIVPF , A.DTH_INI_VIGENCIA , A.NUM_CPF , A.COD_GARANTIA , A.IMP_SEGURADA , A.VLR_PREMIO , A.VLR_TAXA_CALC_PRM , B.NUM_MATRICULA FROM SEGUROS.VG_FUNC_COBERTURA A , SEGUROS.VG_MOV_FUNCIONARIO B WHERE A.NUM_PROPOSTA_SIVPF = '000000000000000' AND A.DTH_INI_VIGENCIA = '2020-01-01' AND A.COD_GARANTIA <> '0000' AND B.NUM_PROPOSTA_SIVPF = A.NUM_PROPOSTA_SIVPF AND B.DTH_INI_VIGENCIA = A.DTH_INI_VIGENCIA AND B.NUM_CPF = A.NUM_CPF AND B.STA_FUNCIONARIO = ' ' ORDER BY A.NUM_PROPOSTA_SIVPF , A.DTH_INI_VIGENCIA , A.COD_GARANTIA , B.NUM_MATRICULA","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"e7cd62e8-7583-439c-b831-86d80ac5fd76","TimeStamp":"2024-12-16T14:47:48.6276465-03:00","Query":"SELECT COD_PESSOA , SEQ_EMAIL , EMAIL , SITUACAO_EMAIL FROM SEGUROS.PESSOA_EMAIL WHERE COD_PESSOA = '000000000' ORDER BY SEQ_EMAIL","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"f93ce596-d396-44c6-8901-d6abebc6cc6e","TimeStamp":"2024-12-16T14:47:49.4152085-03:00","Query":" SELECT RAMO_EMISSOR FROM SEGUROS.APOLICES WHERE NUM_APOLICE = '0000000000000' WITH UR","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"ad063d3e-e1b2-405b-aeb8-3fb09afc76fb","TimeStamp":"2024-12-16T14:47:51.363278-03:00","Query":" SELECT PCT_IOCC_RAMO FROM SEGUROS.RAMO_COMPLEMENTAR WHERE RAMO_EMISSOR = '0000' AND DATA_INIVIGENCIA <= '2020-01-01' AND DATA_TERVIGENCIA >= '2020-01-01' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"35870d3f-88dc-4504-949f-658082d629aa","TimeStamp":"2024-12-16T14:47:52.6757392-03:00","Query":" INSERT INTO SEGUROS.ARQUIVOS_SIVPF VALUES (' ', '0000', '000000000', '2020-01-01', '000000000', '2020-02-01')","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23503] [IBM][DB2] SQL0530N The insert or update value of the FOREIGN KEY \"FK00004\" is not equal to any value of the parent key of the parent table."}
{"DBCallId":"c285958b-d80b-47fb-96c3-b9e0b64d4587","TimeStamp":"2024-12-16T14:47:53.3817379-03:00","Query":" INSERT INTO SEGUROS.ARQUIVOS_SIVPF VALUES (' ', '0000', '000000000', '2020-01-01', '000000000', '2020-02-01')","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23503] [IBM][DB2] SQL0530N The insert or update value of the FOREIGN KEY \"FK00004\" is not equal to any value of the parent key of the parent table."}
{"DBCallId":"1129c6ce-59e6-48b3-a01a-06fe39abc3f5","TimeStamp":"2024-12-16T14:47:54.4381325-03:00","Query":" SELECT COD_PESSOA , CGC , NOME_FANTASIA , COD_USUARIO , TIMESTAMP FROM SEGUROS.PESSOA_JURIDICA WHERE CGC = '000000000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"1309d22f-a9fd-4deb-b1e9-ffc3daa75e11","TimeStamp":"2024-12-16T14:47:55.1539783-03:00","Query":" INSERT INTO SEGUROS.PESSOA VALUES ('000000000', ' ', CURRENT TIMESTAMP, ' ', ' ', NULL)","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23505] [IBM][DB2] SQL0803N One or more values in the INSERT statement, UPDATE statement, or foreign key update caused by a DELETE statement are not valid because the primary key, unique constraint or unique index identified by \"IPGET121\" constrains table \"0000008578\" from having duplicate values for the index key. SQLSTATE=23505"}
{"DBCallId":"a2293ba9-05be-48f6-959a-4eedeede72b3","TimeStamp":"2024-12-16T14:47:56.0046207-03:00","Query":" SELECT VALUE(MAX(COD_PESSOA) ,0) FROM SEGUROS.PESSOA WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"37aae988-be28-4107-a846-29c29af3ad3b","TimeStamp":"2024-12-16T14:47:56.7088469-03:00","Query":" INSERT INTO SEGUROS.PESSOA_JURIDICA VALUES ('001128805', '000000000000000', 'ELIANE FERREIRA ', 'PF0617B ', CURRENT TIMESTAMP)","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23505] [IBM][DB2] SQL0803N One or more values in the INSERT statement, UPDATE statement, or foreign key update caused by a DELETE statement are not valid because the primary key, unique constraint or unique index identified by \"IPGET133\" constrains table \"0000015E09\" from having duplicate values for the index key. SQLSTATE=23505"}
{"DBCallId":"cee975a0-8544-4a2a-9c27-9ecafaac4b2b","TimeStamp":"2024-12-16T14:47:57.4619977-03:00","Query":"SELECT OCORR_ENDERECO , COD_PESSOA , ENDERECO , TIPO_ENDER , BAIRRO , CEP , CIDADE , SIGLA_UF , SITUACAO_ENDERECO FROM SEGUROS.PESSOA_ENDERECO WHERE COD_PESSOA = '000000000' ORDER BY OCORR_ENDERECO","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"f04032a6-83c7-4e1e-98fd-5c68deff4159","TimeStamp":"2024-12-16T14:47:58.1393951-03:00","Query":" SELECT VALUE(MAX(SEQ_EMAIL) ,0) FROM SEGUROS.PESSOA_EMAIL WHERE COD_PESSOA = '000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"0c0033fd-3c79-4ce8-9ee6-9d67528c6dff","TimeStamp":"2024-12-16T14:47:59.8310305-03:00","Query":" INSERT INTO SEGUROS.PESSOA_EMAIL VALUES ('000000000', '0000', ' ', ' ', ' ', CURRENT TIMESTAMP)","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23505] [IBM][DB2] SQL0803N One or more values in the INSERT statement, UPDATE statement, or foreign key update caused by a DELETE statement are not valid because the primary key, unique constraint or unique index identified by \"IPGET128\" constrains table \"000003A768\" from having duplicate values for the index key. SQLSTATE=23505"}
{"DBCallId":"bab161d5-77ee-478a-aee9-f65b22229d26","TimeStamp":"2024-12-16T14:48:00.4736065-03:00","Query":" SELECT COD_PESSOA , NOME_PESSOA , TIPO_PESSOA , TIMESTAMP , COD_USUARIO FROM SEGUROS.PESSOA WHERE COD_PESSOA = '042584833' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"7f41279a-5ba0-4d3f-b80f-ac9723696db9","TimeStamp":"2024-12-16T14:48:01.1031842-03:00","Query":" SELECT VALUE(MAX(SEQ_EMAIL) ,0) FROM SEGUROS.PESSOA_EMAIL WHERE COD_PESSOA = '000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"72668e23-fe9a-4502-9ee3-7401a041d19d","TimeStamp":"2024-12-16T14:48:08.0752969-03:00","Query":" SELECT COD_PESSOA , SEQ_EMAIL , EMAIL , SITUACAO_EMAIL , COD_USUARIO , TIMESTAMP FROM SEGUROS.PESSOA_EMAIL WHERE COD_PESSOA = '000000000' AND SEQ_EMAIL = '0000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"b3252b18-135e-47f2-b86a-87ebfc5f0b5c","TimeStamp":"2024-12-16T14:48:08.9348754-03:00","Query":"SELECT A.COD_AGENCIA , B.COD_FONTE FROM SEGUROS.AGENCIAS_CEF A , SEGUROS.MALHA_CEF B WHERE A.COD_ESCNEG > 99 AND A.COD_ESCNEG < 9999 AND A.COD_ESCNEG = B.COD_SUREG ORDER BY A.COD_AGENCIA","sqlCode":0,"rowsCount":5004,"Time":"0 ms","Error":""}
{"DBCallId":"9b75bcc0-f3ff-4758-bb4f-6838e44b84ba","TimeStamp":"2024-12-16T14:48:09.1660475-03:00","Query":" SELECT VALUE(MAX(OCORR_ENDERECO) ,0) FROM SEGUROS.PESSOA_ENDERECO WHERE COD_PESSOA = '000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"befab6f4-7e3e-4e13-b6fd-3eba5e1f4370","TimeStamp":"2024-12-16T14:48:20.7379799-03:00","Query":" INSERT INTO SEGUROS.PESSOA_ENDERECO VALUES ('000000000', '0000', ' ', '0000', NULL, ' ', '000000000', ' ', ' ', ' ', ' ', CURRENT TIMESTAMP)","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23503] [IBM][DB2] SQL0530N The insert or update value of the FOREIGN KEY \"FK00028\" is not equal to any value of the parent key of the parent table."}
{"DBCallId":"1feaaf09-ca47-4a71-9b59-cbd136bf47b9","TimeStamp":"2024-12-16T14:48:21.5425016-03:00","Query":" SELECT SITUACAO_FONE FROM SEGUROS.PESSOA_TELEFONE WHERE COD_PESSOA = '000000000' AND TIPO_FONE = '0000' AND NUM_FONE = '00000000000' AND DDD = '0000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"a03fa773-7213-48da-b2a2-9c06de09b947","TimeStamp":"2024-12-16T14:48:21.661484-03:00","Query":" SELECT VALUE(MAX(SEQ_FONE) ,0) FROM SEGUROS.PESSOA_TELEFONE WHERE COD_PESSOA = '000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"2ad0e0ed-466e-4f31-95dd-ba5b4eab1755","TimeStamp":"2024-12-16T14:48:21.7880378-03:00","Query":" INSERT INTO SEGUROS.PESSOA_TELEFONE VALUES ('000000000', '0000', '0000', '0000', '0000', '00000000000', ' ', ' ', CURRENT TIMESTAMP)","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23505] [IBM][DB2] SQL0803N One or more values in the INSERT statement, UPDATE statement, or foreign key update caused by a DELETE statement are not valid because the primary key, unique constraint or unique index identified by \"IPGET134\" constrains table \"000000827F\" from having duplicate values for the index key. SQLSTATE=23505"}
{"DBCallId":"c6260080-baef-4a86-9013-66199fe6a424","TimeStamp":"2024-12-16T14:48:21.9060941-03:00","Query":" SELECT COD_PESSOA , COD_RELAC FROM SEGUROS.R_PESSOA_TIPORELAC WHERE COD_PESSOA = '000000000' AND COD_RELAC = '0000' WITH UR","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"b4cc6e36-64db-405f-b419-862994386cad","TimeStamp":"2024-12-16T14:48:22.0282166-03:00","Query":" INSERT INTO SEGUROS.R_PESSOA_TIPORELAC VALUES ('000000000', '0000')","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23503] [IBM][DB2] SQL0530N The insert or update value of the FOREIGN KEY \"FK00002\" is not equal to any value of the parent key of the parent table."}
{"DBCallId":"eeb4a222-6d27-4112-a1f0-76817916e9d6","TimeStamp":"2024-12-16T14:48:22.1421178-03:00","Query":" INSERT INTO SEGUROS.IDENTIFICA_RELAC VALUES ('000000000000000', '000000000', '0000', ' ', CURRENT TIMESTAMP)","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23503] [IBM][DB2] SQL0530N The insert or update value of the FOREIGN KEY \"FK00003\" is not equal to any value of the parent key of the parent table."}
{"DBCallId":"ad714805-5442-4872-9add-fe691fe543d3","TimeStamp":"2024-12-16T14:48:22.2656552-03:00","Query":" SELECT VALUE(MAX(NUM_IDENTIFICACAO) ,0) FROM SEGUROS.IDENTIFICA_RELAC WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"f5f4884e-3b46-4c33-b358-b61f02f45908","TimeStamp":"2024-12-16T14:48:22.399179-03:00","Query":" INSERT INTO SEGUROS.PROPOSTA_FIDELIZ VALUES ('000000000000000', '000000000000000', '0000', '000000000', '000000000000000', '2020-01-01', '0000', '0000', '0000', ' ', '0000', '0000', '0000000000000', '0000', '000.00', '000000000000000', '0000', '0000', '0000000000000', '0000', '00000000000000', ' ', '000000000', '2020-01-01', '0000000000000.00', '0000', '0000000000000.00', '0000.00', '2020-01-01', '0000000000000.00', ' ', ' ', '0000', '000000000', '0000', CURRENT TIMESTAMP, '000000000', '000000000', ' ', ' ', '0000', ' ', '2020-01-01', '000000000', ' ', ' ', NULL, ' ')","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23513] [IBM][DB2] SQL0545N The requested operation is not allowed because a row does not satisfy the check constraint \"PF002C2\"."}
{"DBCallId":"2d52c888-3985-4d1c-8dbc-e29438490f77","TimeStamp":"2024-12-16T14:48:22.4764343-03:00","Query":" INSERT INTO SEGUROS.PROP_SASSE_VIDA VALUES ( '000000000000000', ' ', ' ', ' ', ' ', '0000000000000', '0000', CURRENT TIMESTAMP , NULL , NULL , NULL , NULL )","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23505] [IBM][DB2] SQL0803N One or more values in the INSERT statement, UPDATE statement, or foreign key update caused by a DELETE statement are not valid because the primary key, unique constraint or unique index identified by \"PF005IP\" constrains table \"0000008284\" from having duplicate values for the index key. SQLSTATE=23505"}
{"DBCallId":"a27100e4-b7a6-4294-a660-bb8a7783c8b7","TimeStamp":"2024-12-16T14:48:22.5575881-03:00","Query":" INSERT INTO SEGUROS.HIST_PROP_FIDELIZ VALUES ( '000000000000000' , ' ' , '000000000' , '000000000' , ' ' , ' ' , '000000000' , '0000' , '0000' )","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [42802] [IBM][DB2] SQL0117N The number of values assigned is not the same as the number of specified or implied columns or variables."}